index.bs

```diff
@@ -1548,40 +1567,58 @@ The <dfn constructor for="XRWebGLLayer">XRWebGLLayer(|session|, |context|, |laye
   1. If |context| is lost, throw an {{InvalidStateError}} and abort these steps.
   1. If |context|'s [=XR compatible=] boolean is false, throw an {{InvalidStateError}} and abort these steps.
   1. Initialize |layer|'s {{XRWebGLLayer/context}} to |context|.
-  1. Initialize |layer|'s {{XRWebGLLayer/antialias}} to |layerInit|'s {{XRWebGLLayerInit/antialias}} value.
   1. Initialize |layer|'s {{XRWebGLLayer/ignoreDepthValues}} as follows:
     <dl class="switch">
       <dt> If |layerInit|'s {{XRWebGLLayerInit/ignoreDepthValues}} value is <code>false</code> and the [=XR Compositor=] will make use of depth values
       <dd> Initialize |layer|'s {{XRWebGLLayer/ignoreDepthValues}} to <code>false</code>
       <dt> Otherwise
       <dd> Initialize |layer|'s {{XRWebGLLayer/ignoreDepthValues}} to <code>true</code>
     </dl>
-  1. Initialize |layer|'s {{XRWebGLLayer/framebuffer}} to a new [=opaque framebuffer=] created with |context|.
-  1. Initialize the |layer|'s [=swap chain=].
-  1. If |layer|'s [=swap chain=] was unable to be created for any reason, throw an {{OperationError}} and abort these steps.
+  1. Initialize |layer|'s [=XRWebGLLayer/use default framebuffer=] to |layerInit|'s {{XRWebGLLayerInit/useDefaultFramebuffer}}.
```

Comment:
Let's modify this so that inline sessions cannot accept the creation of XRWebGLLayers with useDefaultFramebuffer set to false.

Reply:
On this week's call we discussed the option to throw an error if an XRLayer is created without composition disabled on an inline session.. I know at the time you were on the fence... What conclusion did you come to?

Reply:
I made it an error. See https://github.com/immersive-web/webxr/pull/656/files#diff-ec9cfa5f3f35ec1f84feb2e59686c34dR1576

Reply:
Apparently I'm incapable of reading ^_^